packages/dashboard/src/project/projectList.tsx

```diff
@@ -15,7 +17,7 @@ const ProjectsList = ({ search }: ProjectsListProps) => {
   const filters: Filters[] = search
     ? [
         {
-          value: null,
+          value: (undefined as unknown) as null,
```

Comment:
This fixes an existing bug which cause searching projects not returning any result.

Reply:
It is not the best way to solve the issue but the easiest. We should make this prop optional for `filters` as may someone need to pass `like` prop not the `value` prop.